src/tree/ResourceGroupTreeItem.ts

```diff
@@ -16,18 +16,18 @@ import { GroupTreeItemBase } from "./GroupTreeItemBase";
 export class ResourceGroupTreeItem extends GroupTreeItemBase {
     public static contextValue: string = 'azureResourceGroup';
 
-    public data?: ResourceGroup;
+    public async getData(): Promise<ResourceGroup | undefined> {
+        return await this.getResourceGroup(this.name);
+    }
+
+    private data?: ResourceGroup;
```

Comment:
This is being set but I don't see it being read anywhere. Were we meaning to use it as a cache so that `getData` only gets called once? If so we need a check+return in there.

Reply:
It's being read in a `refreshImpl` and `get id()`

Reply:
It's `private`, so it shouldn't be. :smile: